src/store/plugins/external-layers.plugin.js

```diff
@@ -62,13 +65,9 @@ async function updatedWMSLayerAttributes(externalLayer, projection) {
         externalLayer.externalLayerId,
         projection,
         externalLayer.opacity,
-        externalLayer.visible
+        externalLayer.visible,
+        false /* ignoreError */
```

Comment:
might be better to tell here that you currently want error to be shown. Even though it's correct that this param is called ignoreError, it could be understood here that you are choosing to ignore errors there when you are instead doing the opposite